datanode/networkhistory/snapshot/service_load_snapshot.go

```diff
@@ -530,7 +532,7 @@ func copyDataIntoDatabase(ctx context.Context, vegaDbConn sqlstore.Connection, c
 		}
 	}
 
-	return totalRowsCopied, nil
+	return totalRowsCopied, fmt.Errorf("it broke son")
```

Comment:
This doesn't look right, you want to always return an error?

Reply:
bollux, that definitely aint right ;-)